src/style.css

```diff
@@ -0,0 +1,19 @@
+
+* {
+  margin: 0;
+}
+
+.redBlock {
+  background-color: red;
+  height: 100px;
+}
+
+.greenBlock {
+  background-color: green;
+  height: 100px;
+}
+
+.blueBlock {
+  background-color: blue;
+  height: 100px;
+}
```

Comment:
all them have similar property, which can be moved to one general class and this class can be added to all the elements that need this style

for example `block`